[2024-10-01T13:09:10.988+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-01T13:09:10.995+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_postgres_connection_dag.validate_data manual__2024-10-01T07:39:00.777881+00:00 [queued]>
[2024-10-01T13:09:10.999+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_postgres_connection_dag.validate_data manual__2024-10-01T07:39:00.777881+00:00 [queued]>
[2024-10-01T13:09:10.999+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-01T13:09:11.026+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-01 07:39:00.777881+00:00
[2024-10-01T13:09:11.037+0530] {standard_task_runner.py:72} INFO - Started process 17767 to run task
[2024-10-01T13:09:11.041+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'test_postgres_connection_dag', 'validate_data', 'manual__2024-10-01T07:39:00.777881+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpz6s31i16']
[2024-10-01T13:09:11.042+0530] {standard_task_runner.py:105} INFO - Job 21: Subtask validate_data
[2024-10-01T13:09:11.084+0530] {task_command.py:467} INFO - Running <TaskInstance: test_postgres_connection_dag.validate_data manual__2024-10-01T07:39:00.777881+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-01T13:09:11.332+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_postgres_connection_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T07:39:00.777881+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T07:39:00.777881+00:00'
[2024-10-01T13:09:11.333+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-01T13:09:11.392+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-01T13:09:11.394+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-01T13:09:11.510+0530] {validator.py:1467} INFO - 	9 expectation(s) included in expectation_suite.
[2024-10-01T13:09:11.664+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/58 [00:00<?, ?it/s]
[2024-10-01T13:09:11.664+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/58 [00:00<?, ?it/s]
[2024-10-01T13:09:11.672+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|3         | 2/58 [00:00<00:00, 246.75it/s]
[2024-10-01T13:09:11.698+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|3         | 2/58 [00:00<00:00, 58.58it/s] 
[2024-10-01T13:09:11.701+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   7%|6         | 4/58 [00:00<00:00, 108.18it/s]
[2024-10-01T13:09:11.728+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   7%|6         | 4/58 [00:00<00:00, 62.90it/s] 
[2024-10-01T13:09:11.731+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  22%|##2       | 13/58 [00:00<00:00, 192.80it/s]
[2024-10-01T13:09:11.756+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  22%|##2       | 13/58 [00:00<00:00, 141.57it/s]
[2024-10-01T13:09:11.785+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  84%|########4 | 49/58 [00:00<00:00, 406.91it/s]
[2024-10-01T13:09:11.785+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  84%|########4 | 49/58 [00:00<00:00, 406.91it/s]
[2024-10-01T13:09:11.796+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  84%|########4 | 49/58 [00:00<00:00, 406.91it/s]
[2024-10-01T13:09:11.804+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 58/58 [00:00<00:00, 406.91it/s]
[2024-10-01T13:09:11.812+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 58/58 [00:00<00:00, 406.91it/s]
[2024-10-01T13:09:11.813+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 58/58 [00:00<00:00, 406.91it/s]
[2024-10-01T13:09:11.813+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 58/58 [00:00<00:00, 389.70it/s]
[2024-10-01T13:09:12.129+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-01T13:09:12.613+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-01T13:09:12.620+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-01T13:09:12.620+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_postgres_connection_dag, task_id=validate_data, run_id=manual__2024-10-01T07:39:00.777881+00:00, execution_date=20241001T073900, start_date=20241001T073910, end_date=20241001T073912
[2024-10-01T13:09:12.662+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-01T13:09:12.676+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-01T13:09:12.691+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
